submissions/description/broken_authentication_and_session_management/privilege_escalation/template.md

```diff
@@ -1,55 +1,62 @@
-<!--
-**Please replace text in each section below**
+# Broken Authentication Allows Privilege Escalation
 
-Authentication Bypass Report
+## Overview of the Vulnerability
 
-Resources:
-- https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/04-Authentication_Testing/04-Testing_for_Bypassing_Authentication_Schema
-- https://www.bugcrowd.com/blog/authentication-bypass/
--->
-Authentication methods can be bypassed through a variety of ways including, calling an internal post authentication page, modifying the given URL parameters, by manipulating the form, or by counterfeiting sessions. A malicious attacker can perform a variety of actions through bypassing authentication methods, and is only limited by the permissions of the privilieged user. This could include viewing/editing senstive customer data, viewing/editing other user permissions, and taking over user accounts. 
+Authentication methods identify and confirm that a user is who they say they are which allows for the appropriate access controls to be given to that user. However, these access controls can be bypassed by a malicious attacker due to broken authentication and broken session management within an application. Authentication and session management controls  can be bypassed through a variety of ways including, calling an internal post authentication page, modifying the given URL parameters, by manipulating the form, or by counterfeiting sessions.
 
-The authentication method for {{application}} at {{url}} is being bypased through {{action}}. This allows a malicious attacker to perform {{action}} which abuses the persmissions of the privilaged user.
+A malicious attacker can perform a variety of actions through bypassing authentication methods and is limited only by the permissions of the user account that they have access to. This could include viewing or editing sensitive customer data, viewing or editing other user permissions, and taking over user accounts. This privilege escalation by the malicious attacker enables them to have access to more resources or functionality within the application.
 
-## Walkthrough & PoC
-<!--
-Provide a step-by-step walkthrough on how to access the vulnerable authentication mechanism and how to exploit the vulnerability to obtain access or perform a function that is intended to enforce authentication.
-Adding a dot-pointed walkthrough with relevant screenshots will speed triage time and result in faster rewards!
--->
+Vulnerability Specifics to the Application:
 
-Example:
+The authentication method for {{application}} at {{url}} is being bypassed through {{action}}. This allows a malicious attacker to perform {{action}} which abuses the permissions of the privileged user.
+
+## Steps to Reproduce
 
-1. Browse to {{url}}/Login and login as a user that is part of the 'cargo' permissions group.
-2. With an Http proxy enabled, click the dropdown at the top left of the page and click 'Freight Control Panel'
```

Comment:
In Markdown, each numbered dot point can be given the same `1.` number, and they will render with the correct numbers. This eases future maintenance of the list.

Reply:
Thank you, I will update and make note